### PR TITLE
[ci] change labels

### PR DESCRIPTION
**Pull request by alvicsam**
Close https://github.com/paritytech/ci_cd/issues/569

### CODE REVIEW

**Comment by the-right-joyce** [comment]
image
`C1-low 📌` should be replaced by: `C1-low`

**Comment by alvicsam** [comment]
Good catch, thanks!

**Review by the-right-joyce — APPROVED** [review]

**Review by rcny — APPROVED** [review]

File: scripts/ci/github/check_labels.sh
Comment (sergejparity) [inline scripts/ci/github/check_labels.sh:29]:
```suggestion
  'C7-critical'
```
Looks like this should be only 'C7-critical' as per table https://www.notion.so/paritytechnologies/3e034a92359149199a33cfe85d17f611?v=54434c7bb30b4952a98280d1ff839249

**Review by sergejparity — CHANGES_REQUESTED** [review]

File: scripts/ci/github/check_labels.sh
Comment (alvicsam) [reply scripts/ci/github/check_labels.sh]: Thanks, also fixed `C7-high` -> `C5-high`

**Review by sergejparity — APPROVED** [review]

**Comment by alvicsam** [comment]
bot rebase

**Review by coderobe — APPROVED** [review]

**Comment by chevdor** [comment]
This PR should NOT be merged until other PRs are ready, especially around the release notes.

It brings major changes in the meaning of critical labels:
- B1
- D1 (major change)
- D2
- D3

More over, one part is missing.
If B7 is removed, what replaces it ?
We have 'B1-note_worthy' which is part of it but we miss the runtime part we talked about.

**Review by chevdor — CHANGES_REQUESTED** [review]
See comments

**Comment by the-right-joyce** [comment]
for now: no changes on the D-Labels, they will come in a separate batch
additionally: please, add a rule to enforce the user to use a X-labels if B1-note_worthy is tagged

**Review by chevdor — APPROVED** [review]
LGTM, before merging though:
- [ ] please make sure to clearly communicate to the Cumulus team that `B7` now becomes `B1` + `X1`
- [ ] we need a PR for the release notes that updates the logic from `B7` to `B1`+`X1`

**Comment by chevdor** [comment]
Please do NOT merge until https://github.com/paritytech/cumulus/pull/1543 is also mergeable and ideally we merge both at the same time.

File: scripts/ci/github/check_labels.sh
Comment (chevdor) [inline scripts/ci/github/check_labels.sh:55]: It would be better to check for the code only: `B1` rather than the label string: `B1-note_worthy`.
Here everything will collapse if we ever rename `B1-note_worthy` to something like `B1-Interesting ℹ️ ` (for whatever other text).

I would suggest adding a function such as:
```
get_code() {
    [[ $1 =~ ^([A-Z][0-9]).* ]] && echo "${BASH_REMATCH[1]}"
}
```
to extract the label "code" and compare only those label codes.

**Review by chevdor — CHANGES_REQUESTED** [review]

File: scripts/ci/github/check_labels.sh
Comment (chevdor) [reply scripts/ci/github/check_labels.sh:55]: It may be also convenient to handle a list of tuples with the code only and the full string:
```
for i in B1,B1-note_worthy X1,X1-whatever; do
    IFS=","
    set -- $i
    echo $1 and $2;
done
```

Comment (alvicsam) [reply scripts/ci/github/check_labels.sh:55]: I guess adding the `get_code()` function requires some refactor. IMO it's not a good option to make this bash-script (or `lib.sh`) more complicated. At this moment I would suggest merging it as is and create an issue for the OpsTooling team to make a proper service for label checks for all 3 repos (cumulus, polkadot and substrate)

Comment (chevdor) [reply scripts/ci/github/check_labels.sh:55]: ok, that works too

**Review by chevdor — APPROVED** [review]

**Pull request merged**
